apps/api/src/app/workflows-v2/usecases/build-step-issues/build-step-issues.usecase.ts

```diff
@@ -0,0 +1,258 @@
+import { Injectable } from '@nestjs/common';
+import { ControlValuesRepository } from '@novu/dal';
+import {
+  ContentIssue,
+  JSONSchemaDto,
+  StepContentIssueEnum,
+  StepIssuesDto,
+  UserSessionData,
+  StepTypeEnum,
+  WorkflowOriginEnum,
+  ControlValuesLevelEnum,
+} from '@novu/shared';
+import {
+  InstrumentUsecase,
+  TierRestrictionsValidateUsecase,
+  TierRestrictionsValidateCommand,
+  dashboardSanitizeControlValues,
+  PinoLogger,
+} from '@novu/application-generic';
+
+import _ from 'lodash';
```

Comment:
Nit picky comment:

```suggestion
import merge from 'lodash/merge';
import capitalize from 'lodash/capitalize';
import isEmpty from 'lodash/isEmpty';
```

This will work much better in a future Novu version where tree shaking will work 100%.